src/components/nostr/Login.tsx

```diff
@@ -402,11 +402,18 @@ function LoggedInUser({ pubkey, onClose }) {
 
 export default function Login() {
   const toast = useToast();
+  const [contacts] = useAtom(contactListAtom);
   const [pubkey, setPubkey] = useAtom(pubkeyAtom);
   const [privkey, setPrivkey] = useAtom(privkeyAtom);
   const { isOpen, onOpen, onClose } = useDisclosure();
   const { t } = useTranslation("common");
 
+  useEffect(() => {
```

Comment:
Is this needed? you can create an account with habla. CC @verbiricha

Reply:
Thanks, @rolznz, for helping with the code review.

I added this code so users don't have to log in every time they access habla.new.

Reply:
This is for read-only access only, right? sounds good!

Reply:
Sure, I got it. I've done it.